morf-core/src/main/java/org/alfasoftware/morf/jdbc/ResultSetIterator.java

```diff
@@ -95,6 +95,38 @@ public ResultSetIterator(Table table, String query, Connection connection, SqlDi
   }
 
 
+  /**
+   * Creates a {@link ResultSetIterator} using the supplied SQL query,
+   * applying no further result sorting beyond that contained in the SQL query.
+   *
+   * @param table Meta data for the query's result.
+   * @param query The query to run.
+   * @param connection The database connection to use.
+   * @param connectionResources The connection resources to use.
+   * @param sqlDialect The vendor-specific dialect for the database connection.
+   */
+  public ResultSetIterator(Table table, String query, Connection connection, ConnectionResources connectionResources, SqlDialect sqlDialect) {

```

Comment:
All of this doesn't need to be duplicated from the other constructor, can we commonalise the code to a single place? I imagined this would be an overloaded constructor that just calls the existing one.